examples/README.md

```diff
@@ -4,12 +4,12 @@ This directory contains the examples for zhinst-toolkit.
 
 We use [jupytext](https://github.com/mwouts/jupytext) to version control our
 examples. Meaning that all our examples are uploaded in markdown, although they
-where written in jupyter notebooks. To convert a example back into a notebook
+are written in jupyter notebooks. To convert a example back into a notebook
 simply do:
 
 ```
 jupytext --to ipynb examples/hf2.md
 ```
 
-We`ve also a skript called [generate_notebooks.sh](generate_notebooks.sh) that
+We`ve also a script called [generate_notebooks.sh](generate_notebooks.sh) that
```

Comment:
wouldn't it make sense to integrate generate_notebooks.sh into generate_notebooks.py?

Reply:
It is integrated, I left out the execute portion of the script as it is not utilized right now.